build/package/Microsoft.DotNet.Cli.Archive.targets

```diff
@@ -20,19 +21,24 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="GenerateArchives" 
-          DependsOnTargets="Init;Layout;SetupGenerateArchivesInputsOutputs" 
-          Inputs="%(GenerateArchivesInputsOutputs.Inputs)" 
+  <Target Name="GenerateArchives"
+          DependsOnTargets="Init;Layout;SetupGenerateArchivesInputsOutputs"
+          Inputs="%(GenerateArchivesInputsOutputs.Inputs)"
           Outputs="%(GenerateArchivesInputsOutputs.Outputs)">
 
-    <ArchiveDirectory
-        FileName="%(GenerateArchivesInputsOutputs.OutFileName)"
-        OutputDirectory="$(ArchiveOutputDirectory)"
-        InputDirectory="%(GenerateArchivesInputsOutputs.InputDirectory)" >
+    <ZipFileCreateFromDirectory
+        Condition=" '$(OSName)' == 'win' "
+        SourceDirectory="%(GenerateArchivesInputsOutputs.InputDirectory)"
+        DestinationArchive="$(ArchiveOutputDirectory)/%(GenerateArchivesInputsOutputs.OutFileName)" />
```

Comment:
How come in Prepare.targets you needed to add the ".zip" extension, but here you don't?